tests/benchmarks/current_consumption/system_off/testcase.yaml

```diff
@@ -48,50 +50,39 @@ tests:
     timeout: 80
   benchmarks.current_consumption.sample.boards.nrf.system_off:
     platform_allow:
-      - nrf54l15dk/nrf54l15/cpuapp
       - nrf52840dk/nrf52840
       - nrf52dk/nrf52832
       - nrf5340dk/nrf5340/cpuapp
-    integration_platforms:
       - nrf54l15dk/nrf54l15/cpuapp
+      - nrf54l15dk/nrf54l10/cpuapp
+      - nrf54l15dk/nrf54l05/cpuapp
+    integration_platforms:
       - nrf52840dk/nrf52840
       - nrf52dk/nrf52832
       - nrf5340dk/nrf5340/cpuapp
+      - nrf54l15dk/nrf54l15/cpuapp
     tags: ppk_power_measure
     harness: pytest
     harness_config:
       fixture: ppk_power_measure
       pytest_root:
         - "${CUSTOM_ROOT_TEST_DIR}/test_measure_power_consumption.py::test_measure_and_data_dump_power_consumption_systemoff"
     timeout: 90
-  benchmarks.current_consumption.sample.boards.nrf.system_off.nrf_retained:
+  benchmarks.current_consumption.sample.boards.nrf.system_off.retained_mem:
     platform_allow:
       - nrf52840dk/nrf52840
       - nrf52dk/nrf52832
-      - nrf5340dk/nrf5340/cpuapp
```

Comment:
nrf5340dk was removed from the platform_allow list. Is this intentional?

Reply:
I have no idea, this is cherry-picked from other PRs. Please ask the relevant contributors.

Reply:
https://github.com/nrfconnect/sdk-nrf/pull/19140/files#r1869874817